cluster-up/cluster/k8s-1.15.1/provider.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+image="k8s-1.15.1:latest"
```

Comment:
to probably want to update this to a shasum :slightly_smiling_face:

Reply:
yep, still need to test if it works properly first.